io.github.jeffshee.Hidamari.json

```diff
@@ -0,0 +1,240 @@
+{
+    "app-id": "io.github.jeffshee.Hidamari",
+    "runtime": "org.gnome.Platform",
+    "runtime-version": "42",
+    "sdk": "org.gnome.Sdk",
+    "command": "hidamari",
+    "finish-args": [
+        "--share=network",
+        "--share=ipc",
+        "--device=dri",
+        "--socket=x11",
+        "--socket=pulseaudio",
+        "--filesystem=home:ro",
```

Comment:
Why is this needed?

Reply:
xdg-picture and maybe xdg-videos should be enough.

Reply:
Hi @JakobDev , thanks for the review.
Which part are you referring to? (it seems normal to me)
Anyway, `socket=x11` is required as the app needs to run in x11. It needs to show a window with a `WINDOW_TYPE_HINT_DESKTOP`, which is not supported in Wayland.
`--filesystem=home:ro` is required to (1) read user video files, and (2) read an HTML file that could be anywhere within the user directory.
`--share=network` is needed for the video streaming function.

Reply:
@hfiguiere Yeah for (1) `xdg-videos` should be enough, but for (2) it isn't. That's why it needs to be `home:ro`.

Reply:
>  Which part are you referring to? (it seems normal to me)

Only the `--filesystem=home` part

> (1) read user video files, and (2) read an HTML file that could be anywhere within the user directory.

That's sounds like the User can pick a File. On Gtk (which looks like you are using) you get the access to the file when the User picks it, so you don't need this.

Reply:
> @hfiguiere Yeah for (1) `xdg-videos` should be enough, but for (2) it isn't. That's why it needs to be `home:ro`.

then xdg-documents.

Reply:
Solved. Now it only use `xdg-videos/Hidamari:create`